### PR TITLE
[doc][yba] CIS hardened OS support

### PR DESCRIPTION
**Pull request by ddhodge**
Server node landing page and SEO
DOC-461

@netlify /preview/yugabyte-platform/prepare/server-nodes/

### CODE REVIEW

**Review by aishwarya24 — APPROVED** [review]
LGTM

File: docs/content/preview/reference/configuration/default-ports.md [hunk 1]
```diff
@@ -72,6 +72,28 @@ The following common ports are required for firewall rules:
...
+Running YugabyteDB on CIS hardened RHEL 8 or 9 requires the following changes to the firewall:
+
+```sh
+#!/bin/bash
+
+sudo dnf repolist
+sudo dnf config-manager --set-enabled extras
+sudo dnf install -y firewalld
+sudo systemctl start firewalld
+
```
Comment (aishwarya24) [inline docs/content/preview/reference/configuration/default-ports.md:86]:
```suggestion
```

File: docs/content/preview/reference/configuration/operating-systems.md [hunk 2]
```diff
@@ -23,7 +23,9 @@ Unless otherwise noted, operating systems are supported by all supported version
 | AlmaLinux 9      | {{<icon/yes>}} | {{<icon/yes>}} |       |
 | Oracle Linux 8   | {{<icon/yes>}} |                | |
 | Red Hat Enterprise Linux 8 | {{<icon/yes>}} |      | Recommended for production |
-| Red Hat Enterprise Linux&nbsp;9.3 and later| {{<icon/yes>}} |  | Supported in v2.20.3 and later.  {{<badge/ea>}} |
+| Red Hat Enterprise Linux 8 CIS Hardened | {{<icon/yes>}} |      | |
```
Comment (aishwarya24) [inline docs/content/preview/reference/configuration/operating-systems.md:26]: any notes to add for Red Hat Enterprise Linux 8 CIS Hardened ? @yushenng . Is it  Recommended for production ?

File: docs/content/preview/yugabyte-platform/prepare/server-nodes-software/software-on-prem-manual.md [hunk 3]
```diff
@@ -142,6 +142,15 @@ Physical nodes (or cloud instances) are installed with a standard AlmaLinux 8 se
 
     Ensure that the `yugabyte` user has permissions to SSH into the YugabyteDB nodes (as defined in `/etc/ssh/sshd_config`).
 
+    If you are using a CIS hardened image and want SSH access to database nodes, you need to manually add the `yugabyte` user to `sshd_config`.
```
Comment (aishwarya24) [inline docs/content/preview/yugabyte-platform/prepare/server-nodes-software/software-on-prem-manual.md:145]: Is this applicable to any CIS hardened image we support - Amazon Linux 2
(ARM, CIS-hardened) and RHEL 8 and 9 ?

**Review by aishwarya24 — APPROVED** [review]
some questions. LGTM overall.

File: docs/content/preview/yugabyte-platform/prepare/server-nodes-software/software-on-prem-manual.md [hunk 4]
```diff
@@ -142,6 +142,15 @@ Physical nodes (or cloud instances) are installed with a standard AlmaLinux 8 se
...
+
+    ```sh
+    add_cmd = (
+            f"sudo sed -i '/^AllowUsers / s/$/ yugabyte/' /etc/ssh/sshd_config && "
```
Comment (chidmuthu) [inline docs/content/preview/yugabyte-platform/prepare/server-nodes-software/software-on-prem-manual.md:149]: I think they have to run this on the node itself? Not sure where this add_cmd is used

**Comment by shashwat-yb** [comment]
Overall looks good to me, please check 2 comments from my side and make any necessary changes required. Thanks!

File: docs/content/preview/yugabyte-platform/prepare/server-nodes-software/software-on-prem-manual.md (hunk 4 above)
Comment (ddhodge) [reply docs/content/preview/yugabyte-platform/prepare/server-nodes-software/software-on-prem-manual.md:149]: @shashwat-yb can you TAL

**Comment by ddhodge** [comment]
> Overall looks good to me, please check 2 comments from my side and make any necessary changes required. Thanks!

Don't see your comments

**Review by shashwat-yb — APPROVED** [review]
LGTM

File: docs/content/preview/yugabyte-platform/prepare/server-nodes-software/software-on-prem-manual.md (hunk 4 above)
Comment (chidmuthu) [reply docs/content/preview/yugabyte-platform/prepare/server-nodes-software/software-on-prem-manual.md:149]: This got copy pasted from itest code, get rid of the add_cmd we only want the actualy linux commands that are run directly on the host.

**Pull request merged**